### PR TITLE
Added sections at the bottom

### PR DESCRIPTION
**Pull request by walkerlj0**

### CODE REVIEW

File: content/en/curriculum/dev-tools/bacalhau/index.md [hunk 1]
```diff
@@ -40,24 +40,36 @@ Compute nodes are responsible for performing the actual computation.
...
 
-Sending your computations and communicating with the Bacalhau network is pretty easy by using the Bacalhau CLI.
-The `bacalhau` command installed on your computer allows you to send a new job (computation), retrieve its status, or get its results.
+### Prerequisites
+You will need to set up an environment to run Bacalhau. There are three different options you can use:
 
-```bash
-> bacalhau docker run ubuntu echo Hello World
-```
+* **Option 1 –** [Install Bacalhau](https://docs.bacalhau.org/getting-started/installation/#installing-the-bacalhau-cli-locally) on your local machine and Install [Python](https://www.python.org/downloads/) & [Pip](https://pip.pypa.io/en/stable/installation/)
```
Comment (mrodriguez3313) [inline content/en/curriculum/dev-tools/bacalhau/index.md:48]:
```suggestion
* **Option 1 –** [Install Bacalhau](https://docs.bacalhau.org/getting-started/installation/#installing-the-bacalhau-cli-locally) on your local machine and install [Python](https://www.python.org/downloads/) & [Pip](https://pip.pypa.io/en/stable/installation/)
```

File: content/en/curriculum/dev-tools/bacalhau/index.md [hunk 2]
```diff
@@ -40,24 +40,36 @@ Compute nodes are responsible for performing the actual computation.
...
 
-The previous command creates a new job in Bacalhau network by using a Docker image.
-Specifically, it runs an Ubuntu container and executes `echo Hello World` inside the container.
+* **Option 2 –** Install [Docker Desktop](https://www.docker.com/products/docker-desktop/) and run [Bacalhau in Docker](https://www.docker.com/products/docker-desktop/)
 
-You can then verify the status of the job by executing the following command.
+* **Option 3 –** Run Bacalhau in [Gitpod](https://gitpod.io/#https://github.com/protocol/launchpad-tutorials) in a web browser
```
Comment (mrodriguez3313) [inline content/en/curriculum/dev-tools/bacalhau/index.md:52]: idk if this link works can you double check it? I wasn't able to get anything to load

File: content/en/curriculum/dev-tools/bacalhau/index.md [hunk 3]
```diff
@@ -40,24 +40,36 @@ Compute nodes are responsible for performing the actual computation.
...
 
-```bash
-> bacalhau list --id-filter=${JOB_ID}
-```
+### Tutorials
+* [Install the Bacalhau Client](https://docs.bacalhau.org/getting-started/installation/#install-the-bacalhau-client) - See instructions for installing Bacalhau on your local machine
+* [Running Oandas in Bacalhau](https://docs.bacalhau.org/examples/workload-onboarding/python-pandas/) – In this tutorial, you will deploy a Python script on Bacalhau. The script a basic application that uses the Pandas Python library
```
Comment (mrodriguez3313) [inline content/en/curriculum/dev-tools/bacalhau/index.md:56]:
```suggestion
* [Running Pandas in Bacalhau](https://docs.bacalhau.org/examples/workload-onboarding/python-pandas/) – In this tutorial, you will deploy a Python script on Bacalhau. The script a basic application that uses the Pandas Python library
```

**Review by mrodriguez3313 — CHANGES_REQUESTED** [review]

File: content/en/curriculum/dev-tools/bacalhau/index.md (hunk 2 above)
Comment (enoldev) [reply content/en/curriculum/dev-tools/bacalhau/index.md:52]: It works for me, it should bring you to Gitpod, where you have to log in

**Comment by walkerlj0** [comment]
Done in another PR

**Pull request closed without merging**